### PR TITLE
Always provide the list of speakers (closes #3003)

### LINKED ISSUE #3003: User without 'agenda.can_see_hidden_items' cannot open list of speakers view of (hidden) motion
To reproduce:
* Create new (empty) group with persmissions "motions.can_see" and "agenda.can_see"
* Create new user and add this new group to user.
* Create new motion.
* Login as user and go to motion detail view.
* Click on "list of speakers" button
=> Failed with "Cannot read property 'id' of undefined" (because the related agenda item is hidden).

Expected: User can see the motion and the agenda, so he should also allowed to see the list of speakers view of this motion.

Is there a possibility to show the list of speakers of such related (hidden) agenda items - also if user has no permission to see hidden items in agenda? A user cannot understand why he can see the motion but not the hidden agenda item of the motion (with its list of speakers view).

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: tests/integration/agenda/test_viewset.py [hunk 1]
```diff
@@ -41,7 +41,13 @@ def test_hidden_by_anonymous_without_perm_to_see_hidden_items(self):
         permission = group.permissions.get(content_type__app_label=app_label, codename=codename)
         group.permissions.remove(permission)
         response = self.client.get(reverse('item-detail', args=[self.item.pk]))
-        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(sorted(response.data.keys()), sorted((
```
Comment (normanjaeckel) [inline tests/integration/agenda/test_viewset.py]: Please add to this test that e. g. duration is not visable.

**Comment by normanjaeckel** [comment]
One remark. Please add CHANGELOG notice that list of speakers for hidden items is now visable. Then rebase and merge.

File: tests/integration/agenda/test_viewset.py (hunk 1 above)
Comment (FinnStutzenstein) [inline tests/integration/agenda/test_viewset.py]: Done.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel Ping.

**Pull request merged**